databricks/koalas/frame.py

```diff
@@ -947,6 +948,63 @@ def groupby(self, by):
         gp = self._sdf.groupby(by)
         return PandasLikeGroupBy(self, gp, None)
 
+    def isin(self, values):
+        """
+        Whether each element in the DataFrame is contained in values.
+
+        :param values: iterable or dict
+                       The sequence of values to test. If values is a dict,
+                       the keys must be the column names, which must match.
+                       Series and DataFrame are not supported.
+        :return: :class:`DataFrame`
+                        DataFrame of booleans showing whether each element
+                        in the DataFrame is contained in values.
+
+        Examples
+        --------
+        When ``values`` is a list:
+
+        >>> df = ks.DataFrame({'A': [1, 2, 3], 'B': ['a', 'b', 'f']})
```

Comment:
why not use the same examples in the pandas doc?

Reply:
I took it from the source code of my Pandas 0.23.4 
I'll change it by the latest one.